test/onnx/onnx_test.cpp

```diff
@@ -1165,6 +1165,40 @@ TEST_CASE(pad_3arg_test)
     EXPECT(p == prog);
 }
 
+TEST_CASE(pad_reflect_test)
+{
+    migraphx::program p;
+    auto l0 = p.add_parameter("0", migraphx::shape{migraphx::shape::float_type, {2, 2}});
+    p.add_literal({migraphx::shape{migraphx::shape::int32_type, {4}}, {0, 2, 0, 1}});
```

Comment:
Why is this added? Its never used.

Reply:
This is because in the latest opset of onnx, the operator takes pads as an input. It is an artifact of parsing in a constant value (see other pad tests).

Reply:
I see.